config/pgi-cxxflags

```diff
@@ -0,0 +1,97 @@
+#							-*- shell-script -*-
+#
+# Copyright by The HDF Group.
+# Copyright by the Board of Trustees of the University of Illinois.
```

Comment:
```suggestion
```
line 4 should be removed since no part of this file existed before 2006.

Reply:
Done